config/routes.rb

```diff
@@ -1,8 +1,24 @@
 Rails.application.routes.draw do
   root to: 'home#index'
 
+  get '/requested_vacations/:user_id',
```

Comment:
Instead of this get you can use member option http://api.rubyonrails.org/classes/ActionDispatch/Routing/Mapper/Resources.html#method-i-member